InfiniTAM/Engine/PoseSourceEngine.cpp

```diff
@@ -2,49 +2,25 @@
 
 #include "PoseSourceEngine.h"
 
-#include "../Utils/FileUtils.h"
-
 #include <stdio.h>
 
-using namespace InfiniTAM::Engine;
-
-PoseSourceEngine::PoseSourceEngine(const char* poseMask) {
-  strncpy(this->poseMask, poseMask, BUF_SIZE);
+#include "../Utils/FileUtils.h"
```

Comment:
The infinitam package is full of such includes...
I will avoid them in my packages